charts/opensearch-operator/values.yaml

```diff
@@ -10,7 +10,9 @@ manager:
       memory: 1Gi
   image:
     repository: public.ecr.aws/opsterio/opensearch-operator
-    tag: latest
+    ## tag default uses appVersion from Chart.yaml, to override specify tag tag: "v1.1"
+    tag: ""
+    pullPolicy: "Always"
```

Comment:
`Always` shouldn't be used in production, use `IfNotPresent` instead to avoid pulling needlessly (saves on bandwith, pull quotas, etc.).

`Always` is only needed when working with development versions where you re-push images with the same tag again.